lisa/tools/make.py

```diff
@@ -84,15 +85,25 @@ def make(
                 update_envs=update_envs,
             )
 
-        # yes '' answers all questions with default value.
-        result = self.node.execute(
-            f"yes '' | make -j{thread_count} {arguments}",
-            cwd=cwd,
-            timeout=timeout,
-            sudo=sudo,
-            shell=True,
-            update_envs=update_envs,
-            expected_exit_code=0,
-            expected_exit_code_failure_message="Failed to make",
-        )
+        if not ignore_error:
```

Comment:
It doesn't need to copy the method. Use default value for expected_exit_code and failure_message.

Reply:
updated